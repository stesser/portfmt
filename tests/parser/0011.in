.for x in 1 2
post-install-$x-on:
.endfor
