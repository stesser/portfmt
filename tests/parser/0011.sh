${PORTFMT} -dd 0011.in | diff -L "0011.expected" -L "0011.actual" -u 0011.expected -
