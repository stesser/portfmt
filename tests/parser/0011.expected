conditional-start           1 .for -
conditional-token           1 .for .for
conditional-token           1 .for x
conditional-token           1 .for in
conditional-token           1 .for 1
conditional-token           1 .for 2
conditional-end             1 .for -
target-start                2 post-install-$x-on post-install-$x-on:
target-end                  2 - -
conditional-start           3 .endfor -
conditional-token           3 .endfor .endfor
conditional-end             3 .endfor -
